web/src/utils/constants.ts

```diff
@@ -77,3 +77,112 @@ export const timeAxisMapping: Record<TimeAverages, keyof Duration> = {
   monthly: 'months',
   yearly: 'years',
 };
+/**
+ * A mapping between the source name and a link to the source.
+ * Sources are coming from zone YAML files (capacity and emission factors) or from parsers (production data).
+ * The links have been collected from various sources and are not guaranteed to be up-to-date.
+ * TODO: In future each link should ideally be stored in the zone configuration YAML files instead of here,
+ * so that the link can be updated along with the source and more easily found and maintained.
+ */
+export const sourceLinkMapping: { [key: string]: string } = {
```

Comment:
Moving [this comment](https://github.com/electricitymaps/electricitymaps-contrib/pull/6789#discussion_r1621407656) by @VIKTORVAV99 :

> Do we want to keep this as a JSON file instead? We keep other config mappings as JSON so it makes sense to do the same here?

I think it would make sense to somehow keep this information available in the zone config files in the future so that you can find the source and a link to it in the same place. This requires some additional work though, so we decided not to do that now. I'd suggest we just keep the map in frontend code like now, instead of doing an intermediate JSON step - and then make a proper YAML setup for it later on :)

Reply:
Once we have it in the YAML I can do a optimized mapping like I did for the contributors 🙌🏻

Reply:
That would be great!